pkg/ingester/ingester_v2.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/validation"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
+	io_prometheus_client "github.com/prometheus/client_model/go"
```

Comment:
If you look at Prometheus sources, this import is named `dto`. Maybe we could keep the same naming for consistency.